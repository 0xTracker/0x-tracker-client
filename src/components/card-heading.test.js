import { render } from 'react-testing-library';
import React from 'react';

import CardHeading from './card-heading';

describe('card heading component', () => {
  it('should render with children', () => {
    const { container } = render(<CardHeading>Hello World</CardHeading>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
