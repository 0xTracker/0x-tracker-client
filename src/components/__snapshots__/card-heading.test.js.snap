// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card heading component should render with children 1`] = `
<h2
  class="card-heading__CardHeading-sc-1332skt-0 fixKTT"
>
  Hello World
</h2>
`;
